### PR TITLE
feat(radio-group): create component

### PR DESCRIPTION
**Pull request by tremblaymath**
Simple component to group radio and simplify usage.
I'll add a variant "button" to implement radio-toggle

### CODE REVIEW

File: packages/oui-radio-group/src/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }));
+
+    const getRadioInputElement = (element) => angular.element(element[0].querySelector("input[type=radio]"));
+    const getRadioInputElementWithValue = (element, value) => angular.element(element[0].querySelector(`input[type=radio][value=${value}]`));
+    const getRadioGroupElement = (element) => angular.element(element[0]);
+
+    describe("Component", () => {
+        describe("attributes", () => {
+
+            it("should assign defined ame to child radios when name attribute is defined", () => {
```
Comment (jimmyfortinx) [inline packages/oui-radio-group/src/index.spec.js]: typo: `ame` > `name`

File: packages/oui-radio-group/src/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                const name = "foo";
+
+                const element = TestUtils.compileTemplate(`<oui-radio-group name=${name}><oui-radio></oui-radio></oui-radio-group>`);
```
Comment (jimmyfortinx) [inline packages/oui-radio-group/src/index.spec.js]: multiple line on this one could be nice

File: packages/oui-radio-group/src/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            it("should add radiogroup role", () => {
+                const element = TestUtils.compileTemplate("<oui-radio-group></oui-radio-group>");
+                $timeout.flush();
+
+                expect(getRadioGroupElement(element).attr("role")).toEqual("radiogroup");
+            });
+        });
+
+        describe("on radio group model change", () => {
+            it("should update child radio models", () => {
```
Comment (jimmyfortinx) [inline packages/oui-radio-group/src/index.spec.js]: You should also test when you change the `ctrl.defaultValue` later. Now you are only testing the first initialisation of the component, but what happen if the model change later ?

```javascript
const ctrl = TestUtils.getController(element);
ctrl.defaultValue = "aValue";
$rootScope.$digest();

expect(..."aValue"...).toEqual(true);
expect(..."bValue"...).toEqual(false);
```

File: packages/oui-radio/src/radio.controller.js [hunk 4]
```diff
@@ -13,11 +14,11 @@ export default class {
     $postLink () {
         // Sometimes the digest cycle is done before dom manipulation,
         // So we use $timeout to force the $apply
-        this.$timeout(() =>
+        this.$timeout(() => {
```
Comment (jimmyfortinx) [inline packages/oui-radio/src/radio.controller.js]: This is equivalent of what was already there.

File: packages/oui-radio-group/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# Radio-group
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+```html
+<oui-radio-group
+  model="..."
+  name="...">
```
Comment (jimmyfortinx) [inline packages/oui-radio-group/README.md]: No `on-change` method on the group itself ? If I want to know when the value has changed I need to add the on-change on each `oui-radio` ?

File: packages/oui-radio-group/src/index.spec.js (hunk 1 above)
Comment (tremblaymath) [reply packages/oui-radio-group/src/index.spec.js]: done

File: packages/oui-radio-group/src/index.spec.js (hunk 2 above)
Comment (tremblaymath) [reply packages/oui-radio-group/src/index.spec.js]: done

File: packages/oui-radio/src/radio.controller.js (hunk 4 above)
Comment (tremblaymath) [reply packages/oui-radio/src/radio.controller.js]: removed

File: packages/oui-radio-group/README.md (hunk 5 above)
Comment (tremblaymath) [reply packages/oui-radio-group/README.md]: done

File: packages/oui-radio-group/README.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    <oui-radio/>
+</oui-radio-group>
+```
+
+## Examples
+
+### Default
+
+```html:preview
+<div ng-init="$ctrl.groupValue = 'second'">
```
Comment (neolitec) [inline packages/oui-radio-group/README.md]: (optional) Hide this element?

**Pull request merged**